examples/generate-go-enums/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,38 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Should be able to render Go Enums and should log expected output to console 1`] = `
+Array [
+  "// Cities represents an enum of string.
+type Cities string
+
+const (
```

Comment:
Nitpick: Would it make sense to add a comment here such as: 
```suggestion
// These are all the valid values for Cities enum.
const (
```

In that way, Godoc documentation will add an extra context, useful IMHO:

```bash 
❯ go doc -all main.go


CONSTANTS

const (
        London    Cities = "London"
        Rome             = "Rome"
        Brusseles        = "Brusseles"
)
    These are all the valid values for Cities enum.


TYPES

type Cities string
    Cities represents an enum of string.
```